release/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id "com.palantir.docker" version "0.25.0"
+}
+
+dependencies {
+    compile project(':situp-core')
+}
+
+task endToEndTests {
+    //dependsOn(':situp-api:test') TODO add E2E and enable when ready
+}
+
+task benchmarkTests {
+    //dependsOn(':situp-api:test') TODO add benchmark test and enable
+}
+
+task buildCore {
+    dependsOn ':situp-core:build'
+}
+
+task dockerPrerequisites {
+    dependsOn 'buildCore'
+    dependsOn 'endToEndTests'
+    dependsOn 'benchmarkTests'
```

Comment:
QUES: Should benchmarkTests be a dependent task for docker build? I think benchmarkTests only measures performance but not fuctionality of SITUP.

Reply:
Good point - for now it looked like we needed it as a prereq for release but as you see its an empty task to show that we can add pre-reqs of that sort if required.